crates/utils/src/env_utils.rs

```diff
@@ -23,3 +24,15 @@ pub fn get_env_var_optional(key: &str) -> Result<Option<String>, VarError> {
 pub fn get_env_car_optional_or_panic(key: &str) -> Option<String> {
     get_env_var_optional(key).unwrap_or_else(|e| panic!("Failed to get env var {}: {}", key, e))
 }
+
+// We default to INFO if the tracing level env is not set properly
+pub fn get_tracing_level_from_string(key: &str) -> Level {
```

Comment:
Level has `from_str`, can we use that?

Reply:
Valid, I overlooked this, Thanks for suggesting, [here's the implementation ](https://github.com/madara-alliance/madara-orchestrator/pull/131/commits/f84763ee50ef7fb2537f96368b76fffda1fb16d1).